h/home.py

```diff
@@ -1,5 +1,14 @@
 """The home/front page of the //hypothes.is/ site."""
 from pyramid import view
+from pyramid import httpexceptions
+
+
+@view.view_config(route_name='via_redirect',
+                  request_method='GET')
+def via_redirect(context, request):
+    url = request.params['url']
```

Comment:
This is a `KeyError` if someone visits without the `url` param. Raise a 404 or perhaps a 400 instead.